### DISCUSSION TITLE
SSL Hybrid Indicator

### DISCUSSION THREAD

**Discussion by Khang152** [DaveSkender/Stock.Indicators#1190] [Ideas]
The SSL Hybrid indicator combines elements of both the SSL Channel and SSL Line indicators, offering a comprehensive view of market strength and trend direction. Its inclusion in the Skender.Stock.Indicators library would greatly enhance the versatility and analytical capabilities of the platform. Did you intend to add this indicator to your library? Thanks

**Comment by DaveSkender** [comment]
It’s not in our roadmap, no.  If you want us to consider it, create an enhancement issue with more information and we’ll evaluate it against other priorities.

**Reply by Khang152** [reply]
Thanks for your reply. I will raise an enhancement issue regarding this indicator.
